universal-application-tool-0.0.1/app/views/admin/questions/QuestionEditView.java

```diff
@@ -243,6 +279,52 @@ private DomContent formQuestionTypeSelect(QuestionType selectedType) {
         .withClasses(Styles.HIDDEN);
   }
 
+  /**
+   * Generate a {@link SelectWithLabel} fixed repeater selector based on the question definition's
+   * repeater.
+   */
+  private SelectWithLabel repeaterOptionWithQuestionDefinition(
+      QuestionDefinition questionDefinition) {
+    Path parentPath = questionDefinition.getPath().parentPath();
+    SimpleEntry<String, String> repeaterPathAndId =
+        new SimpleEntry<>(
+            parentPath.isEmpty() ? NO_REPEATER_DISPLAY_STRING : parentPath.toString(),
```

Comment:
Suggestion (blocking): use the question name rather than the path for the option display value. Rationale is that we don't expose the path anywhere else to the admin so it's unlikely they'll have any means of interpreting it if we display it here.

Reply:
i'm just doing `name` for now. we can make this fancier in the future.